database/factories/PathFactory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Busline;
+use App\Path;
+use Faker\Generator as Faker;
+
+/* @var \Illuminate\Database\Eloquent\Factory $factory */
+$factory->define(Path::class, function (Faker $faker): array {
+    return [
+        'busline_id' => Busline::all()->random()->id,
```

Comment:
Do you know what is the difference between `Busline::all()->random()->id` and `Busline::query()->inRandomOrder()->first()->id`?

Reply:
1. Pulls all the records from the database and then picks one from collection.
2. Shuffles the table index and then pulls single model off the top.
Point taken.

Reply:
Of course in that case it wouldn't be a problem, but it's a common problem all over the internet and I think it's good thing to know. Especially when you're getting a random model from one million models collections.